docs/get-started/add-transformations.md

```diff
@@ -30,11 +30,13 @@ $ cargo build --target wasm32-unknown-unknown --release
 ```
 
 Following the build, the wasm bytecode can be found at `./module/avocado_operations/target/wasm32-unknown-unknown/release/avocado_operations.wasm`.
-If you do not wish to build the wasm file yourself it can be found in `./assets/wasm/avocado_operations.wasm`.
+
```

Comment:
To my mind, either we mention it in this documentation file and we keep it in the repository, OR we remove this section and the file in the repository all together, don't you think?

Reply:
It has been removed in https://github.com/polyphene/getting-started/pull/2